Bottom: 12c335e1f98a56a5d288d40ec05135bfb7af4f43
Top:    a874321c5a65ed5b11b44659c6e3ac440bd2fc8b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-24 02:25:45 +0100

Refresh of tmp

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 6b009e5..520a60a 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1350,7 +1350,7 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
       else
 	{
           /* Need to check if this new hole means we need to kick out any old ones for reprocessing */
-          while (0) {
+          while (1) {
             struct find_inside_info info;
             PLINE *prev;
 
@@ -1370,7 +1370,7 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
             }
 
             printf ("Damnit\n");
-
+#if 1
             /* We need to find the contour before it, so we can update its next pointer */
             prev = container;
             while (prev->next != info.result) {
@@ -1383,6 +1383,7 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
             /* Add hole as the next on the list to be processed in this very function */
             info.result->next = *src;
             *src = info.result;
+#endif
           }
           /* End check for kicked out holes */
